app/components/course_level_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-summary-list__row">
+  <dt class="govuk-summary-list__key">
+    Level
+  </dt>
+  <dd class="govuk-summary-list__value" data-qa="course__level">
+    <%= course.level&.humanize %>
+  </dd>
+  <dd class="govuk-summary-list__actions">
+    <% if changeable %>
+      <%= course_creation_change_button(
```

Comment:
I understand this is using a view helper but couldn't we use bunk standard rails `link_to` instead of having a view helper to output a link.

Reply:
I think this component should ideally be self contained

Reply:
Thanks for your help